changelog/1446/1448

```diff
@@ -0,0 +1,5 @@
+Bugfix: Missing Framework
+
+There was an issue with PocketSVG 2.8.0 that causes all ownBrander builds / clean Xcode builds to fail due to a missing PocketSVG framework, resulting in a crash on startup.
```

Comment:
We should mention that all dependencies are now pinned to an explicit version

Reply:
Done!